streams/src/main/java/org/apache/kafka/streams/processor/internals/PartitionGroup.java

```diff
@@ -211,7 +192,7 @@ public boolean readyToProcess(final long wallClockTime) {
             return false;
         } else {
             enforcedProcessingSensor.record(1.0d, wallClockTime);
-            logger.info("Continuing to process although some partition timestamps were not buffered locally." +
+            logger.trace("Continuing to process although some partitions are empty on the broker." +
```

Comment:
I thought if user did not set to `MAX_TASK_IDLE_MS_DISABLED` they may care about when enforce processing starts, so this is better to stay as INFO?

Reply:
Thanks @guozhangwang, I thought so as well, but @ableegoldman pointed out to me that it actually prints out on every invocation while we are enforcing processing, which turns out to flood the logs.

Reply:
Maybe we could leave this detailed logging at TRACE, and just print a single message at `warn` the first time this enforced processing occurs?